src/Ledger/Utxo.lagda

```diff
@@ -85,11 +85,7 @@ module _ (let open Tx; open TxBody) where
   coinPolicies = policies (inject 1)
 
   isAdaOnlyᵇ : Value → Bool
-  isAdaOnlyᵇ v =
-    if (policies v) ≡ᵉ coinPolicies then
-      true
-    else
-      false
+  isAdaOnlyᵇ v = toBool (policies v ≡ᵉ coinPolicies)
```

Comment:
I missed this in my first review: the inner workings of `toBool` are prohibitively complex for the PDF readers and  that goes against the PDF code being self-contained, hence why I used the Haskell "anti-pattern" `if b then true else false` which linters like `hlint` would report as a code smell that can be fixed by replacing the whole thing with just `b`.

Reply:
Yeah, but the whole property shouldn't be a boolean in the first place. This should all go to `Set` and then this issue will go away (and only `policies v ≡ᵉ coinPolicies` will remain). So this is a step in the right direction, I just didn't want to refactor all that as part of this PR.

Reply:
Oh, I completely agree with that, issue resolved :)